fbpic/openpmd_diag/particle_diag.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, period, species = {"electrons": None}, comm=None,
                 if quantity == "position":
                     self.array_quantities_dict[species_name] += ['x','y','z']
                 elif quantity == "momentum":
-                    self.array_quantities_dict[species_name] += ['ux','uy','uz']
+                    self.array_quantities_dict[species_name] += ['ux','uy','uz','gamma']
```

Comment:
Hm, I think that this changes the default behavior, which is not ideal...
Would you mind, adding an extra `elif quantity == "gamma":`, so that `gamma` is not written when the user enters `"momentum"`, but is only written if the user adds `"gamma"` to the `particle_data` list? (and could you modify the docstring for `particle_data` accordingly`?)

Reply:
Yeah sure, actually the code doesn't need an extra `elif` since the final `else` would already append it.